### PR TITLE
fix typo (solves #60)

### PR DESCRIPTION
**Pull request by 6gk**
Please follow this template, if applicable.

## Description

bugfix: typo

## Usage

usage: "merge pr"

### Showcase

image

## Additional Notes

good pr template 10/10

## Checklist

Please make sure you can check all the boxes that apply to this PR.

- [ ] All widgets I've added are correctly documented.
- [ ] The documentation in the `docs/content/main` directory has been adjusted to reflect my changes.
- [ ] I used `cargo fmt` to automatically format all code before committing

### CODE REVIEW

**Comment by elkowar** [comment]
Doesn't seem to work for me...
image



image

**Comment by 6gk** [comment]
image
aww that sux

**Comment by elkowar** [comment]
DId you consider writing tests for this? Would be cool to avoid future regressions

**Comment by 6gk** [comment]
no, tests are for nerds who aren't confident in writing code that just werks™

File: src/widgets/widget_definitions.rs [hunk 1]
```diff
@@ -191,7 +191,7 @@ fn build_gtk_color_button(bargs: &mut BuilderArgs) -> Result<gtk::ColorButton> {
     let gtk_widget = gtk::ColorButtonBuilder::new().build();
     let on_change_handler_id: Rc<RefCell<Option<glib::SignalHandlerId>>> = Rc::new(RefCell::new(None));
     resolve_block!(bargs, gtk_widget, {
-        // @prop use-alpha - bool to wether or not use alpha
+        // @prop use-alpha - bool to whether or not use alpha
```
Comment (6gk) [inline src/widgets/widget_definitions.rs:194]: :+1: Looks good

File: src/widgets/widget_definitions.rs [hunk 2]
```diff
@@ -191,7 +191,7 @@ fn build_gtk_color_button(bargs: &mut BuilderArgs) -> Result<gtk::ColorButton> {
     let gtk_widget = gtk::ColorButtonBuilder::new().build();
     let on_change_handler_id: Rc<RefCell<Option<glib::SignalHandlerId>>> = Rc::new(RefCell::new(None));
     resolve_block!(bargs, gtk_widget, {
-        // @prop use-alpha - bool to wether or not use alpha
```
Comment (6gk) [inline src/widgets/widget_definitions.rs:194]: Looks bad

File: src/widgets/widget_definitions.rs (hunk 1 above)
Comment (6gk) [inline src/widgets/widget_definitions.rs:194]: Looks good, nice fix. :+1:

**Review by mmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmrn — APPROVED** [review]

**Review by alx365 — APPROVED** [review]

File: src/widgets/widget_definitions.rs (hunk 2 above)
Comment (alx365) [reply src/widgets/widget_definitions.rs:194]: haha

File: src/widgets/widget_definitions.rs (hunk 1 above)
Comment (alx365) [reply src/widgets/widget_definitions.rs:194]: agree ^

**Comment by elkowar** [comment]
I'll still merge this, even without tests,...
Because i trust u, bby <3

**Comment by 6gk** [comment]
thanks for trusting programmers over devs.

**Pull request merged**